docs/src/pages/components/data-grid/columns/ColumnOrderingGrid.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { XGrid } from '@material-ui/x-grid';
+import { useDemoData } from '@material-ui/x-grid-data-generator';
+
+export default function ColumnOrderingGrid() {
+  const { data } = useDemoData({
+    dataSet: 'Employee',
```

Comment:
I would recommend using the same data set as the other demos to not create a distraction. What should be noticed is the feature, not that the data set is different :)

```suggestion
    dataSet: 'Commodity',
```